### DISCUSSION TITLE
Transfer suspension and who can start it again

### DISCUSSION THREAD

**Discussion by ndr-brt** [International-Data-Spaces-Association/ids-specification#254] [Q&A] [answered]
Hi, I have a question about the Transfer "suspension", looks like both Consumer and Provider can send the message to suspend it, my question is, a suspended transfer can be restarted by both of the parts or only by the one who suspended it? 

That's not explicitly listed in the spec but my guess would be that if a Provider suspends a transfer for whatever reason then the Consumer should not be able to start it again. Am I correct?

**Comment by jimmarino** [comment, marked as answer]
I would say this is implementation-specific behavior, but in most cases, only the party that suspended it should likely be able to resume.
